package.json

```diff
@@ -33,7 +33,7 @@
     "serverless-idempotency-helper": "./plugins/serverless-idempotency-helper",
     "serverless-offline": "^6.8.0",
     "serverless-plugin-scripts": "^1.0.2",
-    "serverless-plugin-warmup": "^4.9.0",
+    "serverless-plugin-warmup-helper": "./plugins/serverless-plugin-warmup-helper",
```

Comment:
A custom plugin was written to make the serverless-plugin-warmup idempotent.  We install that here, not the warmup plugin.
** Note, I have a PR open against https://github.com/juanjoDiaz/serverless-plugin-warmup to make the upstream idempotent.  The author says he will merge.  When he takes my code, we can ditch this custom plugin.